demisto_sdk/commands/format/update_generic_yml.py

```diff
@@ -67,6 +67,11 @@ def get_yml_data_as_dict(self):
         print(F'Reading YML data')
 
         with open(self.source_file) as f:
+            # yaml file can not load = with out a value after it -> changed to '='
+            read_file = f.read()
+            replaced = read_file.replace("simple: =", "simple: '='")
```

Comment:
as yml can be also scripts and automations, can we do it only for playbooks?(use the `file_type` func)
@yuvalbenshalom WDYT?